scss/_variables.scss

```diff
@@ -35,7 +35,9 @@ $black: #000 !default;
 
 $blue: #1874dc !default; // Interface, Hyperlinks, UI Button
 $blue-dark: #002856 !default;
+$blue-darkest: #002044 !default;
 $blue-light: #eff5fc !default;
+$blue-lightest: #9EC0E7 !default;
```

Comment:
What if we called `$blue-dark` `$navy` then `$blue-darkest` became `$navy-dark` and `$blue-lightest` became `$navy-light` such that we could delineate between the base of blue (#1874dc) and navy?

Reply:
If you make this change, to prevent breakage, keep `$blue-dark: $navy !default` so apps already using `$blue-dark` don't need an immediate update.

Reply:
I like that solution @sawiggins. @ajswis Will make sure to keep `$blue-dark`.